demos/serverless/template.yaml

```diff
@@ -545,6 +545,20 @@ Resources:
                         "title": "Daily attendees who joined a meeting",
                         "view": "bar"
                     }
+                },
+                {
+                    "type": "log",
+                    "x": 0,
+                    "y": 18,
+                    "width": 24,
+                    "height": 6,
+                    "properties": {
+                        "query": "SOURCE \"${Source}\" | filter name in [\"signalingDropped\"]\n| fields fromMillis(@timestamp) as timestamp,\nconcat(attributes.osName, \" \", attributes.osVersion) as operatingSystem,\nconcat(attributes.browserName, \" \", attributes.browserMajorVersion) as browser,\nattributes.meetingStatus as failedStatus,\nattributes.meetingErrorMessage as reason,\nconcat(attributes.signalingOpenDurationMs / 1000, \"s\")  as signalingOpenDurationMs,\nattributes.retryCount as retryCount\n| sort @timestamp desc\n",
```

Comment:
```
filter name in [\"signalingDropped\"]
```

The SDK does not have the `signalingDropped` event.
https://github.com/aws/amazon-chime-sdk-js/blob/master/src/eventcontroller/EventName.ts

The `signalingDropped` is only available as the meeting history state, so this table won't show any data.
https://aws.github.io/amazon-chime-sdk-js/modules/meetingevents.html#the-meeting-history-attribute

Reply:
Oh I will use the meeting history attribute instead of the meeting event. thanks Kyu!

Reply:
Yon cannot retrieve the value by accessing: `attributes.meetingHistory.signalingDropped`.

`attributes.meetingHistory` is an array of the history state object.

```
[
  {
    name: "audioInputSelected", 
    timestampMs: 1630302522411
  },
  {
    name: "signalingDropped", 
    timestampMs: 1630302522520
  },
  {
    name: "meetingStartRequested", 
    timestampMs: 1630302525540
  },
  {
    name: "signalingDropped", 
    timestampMs: 1630302525600
  }
]
```

You can only access by `attributes.meetingHistory.indexNumber.name`.

```
attributes.meetingHistory.0.timestampMs | 1630302522411
attributes.meetingHistory.1.name | videoInputSelected
attributes.meetingHistory.1.timestampMs | 1630302522520
attributes.meetingHistory.2.name | meetingStartRequested
attributes.meetingHistory.2.timestampMs | 1630302525540
attributes.meetingHistory.3.name | videoInputUnselected
attributes.meetingHistory.3.timestampMs | 1630302525600
attributes.meetingHistory.4.name | meetingStartFailed
attributes.meetingHistory.4.timestampMs | 1630302526467
```

Once we publish the `signalingDropped` as a meeting event, we can use your previous solution to count or list all `signalingDropped` evens. Let's revisit this PR later.